enterprise/core-edge/src/main/java/org/neo4j/coreedge/discovery/HazelcastClient.java

```diff
@@ -72,7 +72,7 @@ public ClusterTopology currentTopology()
         {
             log.info( "Failed to read cluster topology from Hazelcast. Continuing with empty (disconnected) topology. "
                     + "Connection will be reattempted on next polling attempt.", e );
-            return new ClusterTopology( false, emptyMap(), emptySet() );
+            return new ClusterTopology( null /* TODO */, false, emptyMap(), emptySet() );
```

Comment:
It will sort itself out in the next couple of days by Magic™. But to be real, the entire binding of edges isn't present, so there is really no natural way to express how much is lacking. Just Let It Be™.